Assets/Scripts/PickupAmmunition.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+
+public class PickupAmmunition : MonoBehaviour
```

Comment:
nit: I find it difficult to see the distinction in the names `Ammunition` and `PickupAmmunition`. Suggest something like this (especially trying to keep the `Player-` prefix on player components):

- `Ammunition` &rarr; `Pickupable` (or keep `Ammunition`, it's quite good)
- `PickupAmmunition` &rarr; `PlayerInventory` or `PlayerWeapon`